### DISCUSSION TITLE
Vehicle_location problem?

### DISCUSSION THREAD

**Discussion by FilipDem** [bimmerconnected/bimmer_connected#694] [Q&A]
Has anybody seen something strange with the vehicle (https://github.com/bimmerconnected/bimmer_connected/blob/ce073425b4030a22a52f3a1c585480eb3d44e45a/bimmer_connected/vehicle/vehicle.py#L89).
I am using this to define if the car is at home or not... However in lot of case this returns `None`. However when looking it up into the data (https://github.com/bimmerconnected/bimmer_connected/blob/master/bimmer_connected/vehicle/vehicle.py#L224) after the update, I see the location is available...

Location of vehicle: 
* `VehicleLocation(location=None, heading=None, vehicle_update_timestamp=None, account_region=<Regions.REST_OF_WORLD: 'row'>, remote_service_position=None)` 
* workaround `self.myVehicle.data[ATTR_STATE].get('location')` gives `{'coordinates': {'latitude': xx.94467972222222, 'longitude': yy.3341030555555555}, 'address': {'formatted': 'xxxxxxxxxx'}, 'heading': 310})`

Somebody has noticed something similar?

Filip

**Comment by rikroe** [comment]
Could somehow be related to #692, as there are some check against timestamps?

**Comment by FilipDem** [comment]
Well, I already had a look before, but didn't notice any errors so I couldn't link this to https://github.com/bimmerconnected/bimmer_connected/issues/692. However I tested the commit and also the location seems to work now! Thanks.
